PyInstaller/depend/utils.py

```diff
@@ -329,6 +329,9 @@ def _restorePaths(old):
         # 'W: library coredll.dll required via ctypes not found'
         if cbin in ['coredll.dll', 'kernel32.dll']:
             continue
+        # On unix ignore the libc
+        if is_unix and cbin in ['libc.so', 'libc.so.6']:
```

Comment:
This should be
`cbin == 'libc.so' or cbin.startswith('libc.so.'):`
Otherwise we would run into problems again for e.g. `libc.so.7`.